components/Antenna.tsx

```diff
@@ -0,0 +1,45 @@
+import React from "react"
+import { Alert, Platform, TouchableOpacity, View } from "react-native"
+import { FontAwesome5, MaterialCommunityIcons } from '@expo/vector-icons'; 
+import { SauseeState } from "../shared/TypeDefinitions";
+import { connect, ConnectedProps } from "react-redux";
+import { setUseLocalTiles} from "../shared/ActionCreators";
+
+
+const mapStateToProps = (state: SauseeState) => {
+  return {
+    isUsingLocalTiles: state.isUsingLocalTiles,
+  };
+}
+
+const connector = connect(mapStateToProps, { setUseLocalTiles });
+
+type AntennaProps = ConnectedProps<typeof connector>
+
+const Antenna = (props: AntennaProps) => {
+  return (
+    <View style={{marginRight: Platform.OS == "ios" ? 20 : 10}}>
+      {props.isUsingLocalTiles && <View style={{position: "absolute"}}><FontAwesome5 name="slash" size={20} color="black" /></View>}
+      <TouchableOpacity
+        //disabled={true}
+        onPress={() => {
+          Alert.alert(
+            `Bruke ${props.isUsingLocalTiles ? "internettkart" : "lokalt nedlastet kart"} istedenfor?`,
+            `Dette kan føre til at kartet du trenger ikke vises på skjermen. ${props.isUsingLocalTiles ? "" : "Kartet som du midlertidig har lasted ned vil forsvinne."}`,
```

Comment:
Forslag til formulering ved bytting fra nedlastet kart til internettkart:

> Dersom du ikke har mobildekning kan dette kan føre til at kartet du trenger ikke vil vises

Høres litt omvendt ut at når du går online så risikerer du at du ikke får opp kartet, så forslaget til formulering er for å spesifisere at det kun er manglende mobildekning som potensielt gjør at kartet ikke vises. Og så det korresponderer det bedre med meldingen som gis ved skipping av nedlasting i DMS

Og forslag til formulering ved bytting tilbake:

> Bare de områdene av kartet som du har lastet ned vil være synlig

Jeg tenker at bøndene forbinder "laste ned kart" med nedlastingen som de gjør i DMS og ikke det at appen cacher internettkart, så jeg tenker formuleringen bør matche den forståelsen

Reply:
local -> cloud
Dersom du ikke har mobildekning kan dette kan føre til at kartet du trenger ikke vil vises

cloud -> local
Bare de områdene av kartet som du har lastet ned vil være synlig. Hvis du vil bruke internettkart igjen, må kartet hentes inn på nytt.